Cargo.lock

```diff
@@ -60,12 +60,6 @@ version = "1.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa"
 
-[[package]]
```

Comment:
(same comment as in `crypto_core`): we should remove Cargo.lock as it is not recommended for a library: see answer of `alexcrichton` https://github.com/rust-lang/cargo/issues/315#issuecomment-51075730

This way, dependencies are not locked and are not scanned/reported by Github Security Alert